proto/rpc.proto

```diff
@@ -6,15 +6,55 @@ import "requests.proto";
 import "responses.proto";
 
 service Api {
+    // Gets a list of proofs for given nullifier hashes, each proof as a sparse Merkle Tree.
     rpc CheckNullifiers(requests.CheckNullifiersRequest) returns (responses.CheckNullifiersResponse) {}
+
+    // Returns a list of nullifiers that match the specified prefixes and are recorded in the node.
     rpc CheckNullifiersByPrefix(requests.CheckNullifiersByPrefixRequest) returns (responses.CheckNullifiersByPrefixResponse) {}
+
+    // Returns the latest state of an account with the specified ID.
     rpc GetAccountDetails(requests.GetAccountDetailsRequest) returns (responses.GetAccountDetailsResponse) {}
+
+    // Returns the latest state proofs of accounts with the specified IDs.
     rpc GetAccountProofs(requests.GetAccountProofsRequest) returns (responses.GetAccountProofsResponse) {}
+
+    // Returns delta of the account states in the range from `from_block_num` (exclusive) to
+    // `to_block_num` (inclusive).
     rpc GetAccountStateDelta(requests.GetAccountStateDeltaRequest) returns (responses.GetAccountStateDeltaResponse) {}
```

Comment:
Not for this PR, but it feels like we should normalize these endpoints somehow. For example:

- I'm not sure if we are actually using `GetAccountStateDelta` in the client - @igamigo, could you check?
- I think the current `GetAccountDetails` endpoint returns the entire account, which would probably not work for large accounts.
- `GetAccountProofsRequest` is used primarily for FPI, but maybe it can be used for other things as well?

Generally, we should probably list the use case we have for retrieving account data from the node and see if these 3 methods are still the right way to get this data. @polydez - could you create an issue for this?